atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -64,10 +64,8 @@
 class AddLabelsPlugin(PreBuildPlugin):
     key = "add_labels_in_dockerfile"
     is_allowed_to_fail = False
-    rewrite_whitelist = ['release', 'version']
 
-    def __init__(self, tasker, workflow, labels,
-                 dont_overwrite=("Architecture", "architecture"),
+    def __init__(self, tasker, workflow, labels, dont_overwrite=None,
```

Comment:
well I wanted to make it consistent with others, because we do it like this always